sdk/monitor/monitor-ingestion/src/utils/concurrentPoolHelper.ts

```diff
@@ -13,14 +13,13 @@ export async function concurrentRun<T>(
   const promises: Array<Promise<void>> = [];
 
   function removePromise(p: Promise<void>): void {
-    promises.splice(promises.indexOf(p), 1);
+    void promises.splice(promises.indexOf(p), 1);
```

Comment:
why is this needed now?

Reply:
It was an ESLint error where it whether it was intentional or not to not away removing a promise and not awaiting it.  Possible leaving dangling promises.  By adding `void` we tell the compiler we don't care.

Reply:
TIL. I used to always add eslint-disable-next-line for this.